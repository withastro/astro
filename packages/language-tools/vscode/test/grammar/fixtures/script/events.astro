<button onclick="(hello) => {console.log('hello')};">Click Me!</button>

<button onclick='(hello) => {console.log("hello")};'>Click Me!</button>
