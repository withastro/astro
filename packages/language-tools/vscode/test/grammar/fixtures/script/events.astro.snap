><button onclick="(hello) => {console.log('hello')};">Click Me!</button>
#^ source.astro meta.scope.tag.button.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^^^^ source.astro meta.scope.tag.button.astro meta.tag.start.astro entity.name.tag.astro
#       ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro
#        ^^^^^^^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro entity.other.attribute-name.astro
#               ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro punctuation.separator.key-value.astro
#                ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro meta.embedded.line.js string.quoted.astro punctuation.definition.string.begin.astro
#                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro meta.embedded.line.js string.quoted.astro
#                                                   ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro meta.embedded.line.js string.quoted.astro punctuation.definition.string.end.astro
#                                                    ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#                                                     ^^^^^^^^^ source.astro text.astro
#                                                              ^^ source.astro meta.scope.tag.button.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#                                                                ^^^^^^ source.astro meta.scope.tag.button.astro meta.tag.end.astro entity.name.tag.astro
#                                                                      ^ source.astro meta.scope.tag.button.astro meta.tag.end.astro punctuation.definition.tag.end.astro
>
><button onclick='(hello) => {console.log("hello")};'>Click Me!</button>
#^ source.astro meta.scope.tag.button.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^^^^ source.astro meta.scope.tag.button.astro meta.tag.start.astro entity.name.tag.astro
#       ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro
#        ^^^^^^^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro entity.other.attribute-name.astro
#               ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro punctuation.separator.key-value.astro
#                ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro meta.embedded.line.js string.quoted.astro punctuation.definition.string.begin.astro
#                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro meta.embedded.line.js string.quoted.astro
#                                                   ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro meta.attribute.onclick.astro meta.embedded.line.js string.quoted.astro punctuation.definition.string.end.astro
#                                                    ^ source.astro meta.scope.tag.button.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#                                                     ^^^^^^^^^ source.astro text.astro
#                                                              ^^ source.astro meta.scope.tag.button.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#                                                                ^^^^^^ source.astro meta.scope.tag.button.astro meta.tag.end.astro entity.name.tag.astro
#                                                                      ^ source.astro meta.scope.tag.button.astro meta.tag.end.astro punctuation.definition.tag.end.astro
>